planner/core/testdata/enforce_mpp_suite_in.json

```diff
@@ -113,5 +113,15 @@
       "EXPLAIN select a, count(b), avg(distinct c), count(distinct c) from t group by a; -- multi distinct funcs, bail out",
       "EXPLAIN select count(b), count(distinct c) from t; -- single distinct func but no group key, bail out"
     ]
+  },
+  {
+    "name": "TestMPPSingleDistinct3Stage",
+    "cases": [
+      "set @@tidb_allow_mpp=1;set @@tidb_enforce_mpp=1;",
+      "EXPLAIN select count(distinct c) from t;",
```

Comment:
Why does the second stage in explain show count(distinct c) instead of group by c?
(json files cannot be commented, so I put the comments here)

Reply:
See here:
```
e.g.
explain select count(distinct a), count(b) from foo

  HashAgg sum(#1), sum(#2)                              -> final agg
   +- Exchange Passthrough
       +- HashAgg count(distinct a) #1, sum(#3) #2      -> middle agg
          +- Exchange HashPartition by a
               +- HashAgg count(b) #3, group by a       -> partial agg
                   +- TableScan foo
```

After the 1st stage agg, we shuffle by the distinct key. So it is ok to compute count(distinct) in 2nd stage.